com.unity.multiplayer.mlapi/Tests/Editor/Timing/NetworkTimeTests.cs

```diff
@@ -0,0 +1,226 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using MLAPI.Timing;
+using NUnit.Framework;
+using UnityEngine;
+using Random = System.Random;
+
+namespace MLAPI.EditorTests.Timing
+{
+    public class NetworkTimeTests
```

Comment:
wouldn't we need multi-instance or multi-process tests with host-clients & server-clients setups? I'd like to see time & tick being utilized and synced between host/server and clients therefore tested in automated tests. what would you think?

Reply:
For now I just have unit tests which ensure that the individual pieces of the time system behave correctly. E.g I test whether the client speeds up or slows down if latency changes etc. I do not have any end-to-end multi-instance tests.

The reason for that is, is because the time system is not properly coupled to MLAPI yet so most likely these end-to-end tests will not perform as expect. Maybe I should still write a multi-instance test but comment it out for now?

Reply:
There is a multi instance test now testing 2 clients connected to a host with different tick/frame rates.